### PR TITLE
Filipino Translation for Libgit2 - translated 1,447 words

### PR DESCRIPTION
**Pull request by dranem**
translated 1,447 words from english to tagalog

### CODE REVIEW

File: book/B-embedding-git/sections/libgit2.asc [hunk 1]
```diff
@@ -65,12 +62,12 @@ puts "#{commit.author[:name]} <#{commit.author[:email]}>"
 tree = commit.tree
 ----
 
-As you can see, the code is much less cluttered.
-Firstly, Rugged uses exceptions; it can raise things like `ConfigError` or `ObjectError`  to signal error conditions.
-Secondly, there's no explicit freeing of resources, since Ruby is garbage-collected.
-Let's take a look at a slightly more complicated example: crafting a commit from scratch
+Gaya ng makikita mo, ang code ay cluttered na masmababa.
```
Comment (heywens) [inline book/B-embedding-git/sections/libgit2.asc]: two words have been combined. please add space character > ``mas mababa``.

File: book/B-embedding-git/sections/libgit2.asc [hunk 2]
```diff
@@ -32,31 +32,28 @@ git_repository_free(repo);
...
 
 Ang pangalawang tipak ng code ay gumagamit ng rev-parse na syntax (tingnan ang <<_git_tools#_branch_references>> para sa higit pa nito) upang makuha ang commit na ang HEAD na sa kalaunan.
-Ang binalik na uri ay isang `git_object` pointer, na kumakatawan sa isang bagay na umiiral sa database ng Git object para sa isang repository.
-`git_object` is actually a ``parent'' type for several different kinds of objects; the memory layout for each of the ``child'' types is the same as for `git_object`, so you can safely cast to the right one.
-In this case, `git_object_type(commit)` would return `GIT_OBJ_COMMIT`, so it's safe to cast to a `git_commit` pointer.
+Ang binalik na uri ay isang `git_object` pointer, na kumakatawan sa isang bagay na umiiral sa database ng Git object para sa isang repository. `git_object` ay talagang isang ``parent'' na uri para sa maraming iba't ibang mga uri ng mga bagay; ang layout ng memorya para sa bawat isa sa mga uri ng ``child'' ay kapareho ng para sa `git_object`, para maaari mong isumite ng ligtas ang tama. Sa kasong ito, ang `git_object_type(commit)` ay babalik sa `GIT_OBJ_COMMIT`, kaya ligtas itong isumite sa isang `git_commit` pointer.
```
Comment (heywens) [inline book/B-embedding-git/sections/libgit2.asc]: ``iba't ibang`` to ``iba't-ibang``

File: book/B-embedding-git/sections/libgit2.asc [hunk 3]
```diff
@@ -32,31 +32,28 @@ git_repository_free(repo);
...
 
-* If you declare a pointer and pass a reference to it into a Libgit2 call, that call will probably return an integer error code.
-  A `0` value indicates success; anything less is an error.
-* If Libgit2 populates a pointer for you, you're responsible for freeing it.
-* If Libgit2 returns a `const` pointer from a call, you don't have to free it, but it will become invalid when the object it belongs to is freed.
-* Writing C is a bit painful.
+* Kung ipinapahayag mo ang isang pointer at magpasa ng isang reperensiya nito sa isang Libgit2 call, ang tawag na iyon ay malamang na babalik ang isang error code ng integer.
+  Ang halaga na `0` ay nagpapahiwatig ng tagumpay; Ang anumang mas mababa ay mali.
+* Kung pinarami ang pointer ng Libgit2 para sa iyo, ikaw ay responsable para sa pagbabakanti nito.
+* Kung ang Libgit2 ay magbabalik ng isang `const` na pointer mula sa isang tawag, hindi mo kailangang palayain ito, ngunit magiging imbalido kapag ang bagay nabibilang nito ay napalaya.
```
Comment (heywens) [inline book/B-embedding-git/sections/libgit2.asc]: > kapag ang bagay na kinabibilangan nito ay napalaya

**Review by heywens — COMMENTED** [review]
minor reviews

File: book/B-embedding-git/sections/libgit2.asc (hunk 1 above)
Comment (dranem) [reply book/B-embedding-git/sections/libgit2.asc]: changed to mas mababa

File: book/B-embedding-git/sections/libgit2.asc (hunk 2 above)
Comment (dranem) [reply book/B-embedding-git/sections/libgit2.asc]: done adding -

File: book/B-embedding-git/sections/libgit2.asc (hunk 3 above)
Comment (dranem) [reply book/B-embedding-git/sections/libgit2.asc]: Done changing it to "kinabibilangan". Thanks for the review @heywens

**Pull request merged**